### ISSUE TITLE
edit script doesn't scroll down to latest log message

### ISSUE THREAD

**Issue by d4r3topk** [societyserver/sTeam#20]

**Comment by ajinkya007** [comment]
is this issue open? and what are the open issues in this repo?

**Comment by d4r3topk** [comment]
Yes this is open. For extra info : I have used a golden ratio script and a
auto scroll script there. It will be best if you could remove those and
make it work without those. You need some knowledge in emacs and vim. It is
not that difficult (Don't bother going through the golden ratio script, I
haven't written it).

All the issues are open, but don't pick any issues with the labels : coal,
module, core-server. Look at tab completion and edit. edit is your best
chance to solve.

**Comment by ajinkya007** [comment]
where is the edit script? I ain't familiar with the application structure as of now.

**Comment by d4r3topk** [comment]
All the scripts are in the tools directory of sTeam. (Atleast the ones that
you will be dealing with)

**Comment by ajinkya007** [comment]
PR #48 solves this.
